pages/index.ejs

```diff
@@ -1,46 +1,47 @@
 ---
 title: Privacy Manager
-description: Get your webiste up and running online in no time using current starter
-header: <fix>CMintS</fix> starter
+description: Privacy Manager website
 ---
 
 <section class="col2 container">
   <div>
     <h2>{privacy-manager-header[Header of the section] Privacy Manager}</h2>
```

Comment:
I guess this is done too.

Reply:
I can't see this StringID is changed.

Reply:
You mean to use `privacy-manager` instead of `privacy-manager-header` here?

Reply:
> You mean to use privacy-manager instead of privacy-manager-header here?

Well, the meeting-note says: `Let's use overal meaning of the heading for creating i18n strings`, so basically I can't see this row touched and no explanation why.

Reply:
In comparison current string id contain meaning+location.